example123_test.go

```diff
@@ -0,0 +1,31 @@
+//go:build go1.23
+// +build go1.23
```

Comment:
I think this syntax is deprecated (at least in go 1.23). The line before is enough